docs/modules/ROOT/pages/microprofile-certification/5.36.0/README.adoc

```diff
@@ -0,0 +1,9 @@
+= MicroProfile 4.1 Certification Summaries
+
+Payara Server is certified https://jakarta.ee/[MicroProfile 4.1] implementation.
+As proof of compatibility, output of Jakarta EE Compatibility Test Suite tests are presented for each certified version.
+
+The following distributions of Payara 5.36.0 are certified, with results for each listed below:
+
+== Payara 5.36.0 Full Profile
+* xref:jakartaee-certification/5.36.0/microprofile-4.1-tck-results-full-5.36.0.adoc[Payara Server 5.36.0 - Full profile]
```

Comment:
Netlify preview indicates a wrong link for `Payara Server 5.36.0 - Full profile`